### PR TITLE
fix: add warn where trigger_value is below 0.1 grt

### LINKED ISSUE #323: Create warning for too low trigger value
Some indexers are setting the trigger value too low, we consider that it must be at least 0.1 GRT, ideally over 1 GRT. Anything lower can cause issues with the heaviest allocation not returning any receipts.

We should add a warning in case it's lower than 0.1 GRT.

### PR DESCRIPTION
**Pull request by carlosvdr**
fixes #323

### CODE REVIEW

File: config/src/config.rs [hunk 1]
```diff
@@ -94,6 +94,28 @@ impl Config {
...
+            < minimum_recommended_for_max_willing_to_lose_grt
+                .to_u128()
+                .unwrap()
+        {
+            warn!(
+                "Trigger value is too low,currently below 0.1 GRT. \
+                Please modify `max_amount_willing_to_lose_grt` or `trigger_value_divisor`.
+                To have a higher trigger value, ideally above 1 GRT \
+                Anything lower can cause issues with the heaviest \
+                allocation not returning any receipts.
```
Comment (gusinacio) [inline config/src/config.rs:114]: The message could be: the system may constantly deny the sender.

File: config/src/config.rs [hunk 2]
```diff
@@ -94,6 +94,28 @@ impl Config {
...
+            .to_u128()
+            .expect("Could not represent the trigger value in u128");
+        let minimum_recommended_for_max_willing_to_lose_grt = 0.1;
+        if trigger_value
+            < minimum_recommended_for_max_willing_to_lose_grt
+                .to_u128()
+                .unwrap()
+        {
+            warn!(
+                "Trigger value is too low,currently below 0.1 GRT. \
```
Comment (gusinacio) [inline config/src/config.rs:110]: typo, space between comma and next letter

File: config/src/config.rs [hunk 3]
```diff
@@ -94,6 +94,28 @@ impl Config {
...
+                Please modify `max_amount_willing_to_lose_grt` or `trigger_value_divisor`.
+                To have a higher trigger value, ideally above 1 GRT \
```
Comment (gusinacio) [inline config/src/config.rs:112]: could you review the punctuation and capitalization of letters?

**Review by gusinacio — CHANGES_REQUESTED** [review]

**Comment by gusinacio** [comment]
Also, technically this is a feature. We are not fixing something that is broken, we are adding an alert functionality.

**Comment by carlosvdr** [comment]
Hmm i dont like it being a feature but its closer to that rather than a fix yes

**Review by gusinacio — APPROVED** [review]
LGTM

**Pull request merged**